flask_bootstrap/templates/base/table.html

```diff
@@ -154,11 +154,9 @@
                   {{ raise('You have to enable the CSRFProtect extension from Flask-WTF to use delete_url, see the docs for more details (https://bootstrap-flask.readthedocs.io/en/stable/macros.html#render-table).') }}
                   {% endif %}
                 <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
-                <a class="action-icon text-decoration-none"
-                    href="javascript:{}"
-                    onclick="this.closest('form').submit();return false;"
-                    title="{{ config['BOOTSTRAP_TABLE_DELETE_TITLE'] }}">
+                <button type="submit" class="btn btn-link p-0 border-0" title="{{ config['BOOTSTRAP_TABLE_DELETE_TITLE'] }}">
                     {{ render_icon('trash-fill') }}
+                  </button>
                 </a>
```

Comment:
The closing `a` tag should be removed.

@PanderMusubi Can you help to verify if the button change here is working as expected?

Reply:
Removed.

Reply:
They should be on the same height now.

Reply:
Good job!